report_thesis/src/sections/proposed_approach/proposed_approach.tex

```diff
@@ -5,9 +5,55 @@ \section{Proposed Approach}
+The notion of using multiple models per oxide is supported by the advent of models such as the \gls{moc}~\cite{cleggRecalibrationMarsScience2017} model, which combines the predictions of multiple models using a predetermined weighting for each model's predictions on a per-oxide basis.
+While this approach improved accuracy compared to individual models, it required manual tuning of the weights for each model.
+This manual tuning presents limitations, including the analysis required to determine appropriate weights and the risk of suboptimal weighting.
+Given these limitations, it is reasonable to explore techniques that can automate the weighting process while still leveraging the strengths of multiple models.
+To fulfill these criteria, we chose to adopt a stacking ensemble approach. 
+Stacking, as described in Section~\ref{subsec:stacked-generalization}, is a method that utilizes multiple base estimators trained on the same data, whose predictions are then used to train a meta-learner.
+By combining a diverse set of base models, stacking can correct for the biases of individual models.
+Since each model focuses on different patterns within the data, stacking mitigates the inherent biases of individual models by estimating and correcting for these biases.
+Leveraging the strengths of multiple models that each approach the problem differently can lead to better generalization on unseen data.
+This is achieved by using a meta-learner to discern patterns in the base predictors' outputs\cite{wolpertstacked_1992, survey_of_ensemble_learning}, with the added benefit of automating and potentially improving upon the manual tuning employed by the \gls{moc} model.
+However, it is crucial to consider the training of the base models to prevent data leakage and overfitting.
+As emphasized by \citet{cvstacking}, if the base models are trained on the same dataset, the meta learner might favor certain base models over others.
+This can cause the meta learner to be influenced by the same patterns and biases that the base models are susceptible to, leading to overfitting.
+To mitigate this risk and ensure generalizability, a cross-validation strategy should be employed to ensure that the meta learner's training data accurately reflects the true performance of the base learners.
+
+We adopted an experimental approach to empirically evaluate the potential of various models and preprocessing techniques for use in our stacking ensemble. 
+This ensured our selections were informed by the literature review while allowing for independent assessment and validation.
+
+To systematically address the challenges in predicting major oxide compositions from \gls{libs} data, we have devised a comprehensive approach that integrates model and preprocessing selection, an experimental framework, evaluation and comparison, and the construction of a stacking ensemble.
```

Comment:
Don't like the use of the adjective "comprehensive" here - an approach can't be comprehensive IMO.

also adjective bad

Reply:
```suggestion
To systematically address the challenges in predicting major oxide compositions from \gls{libs} data, we have devised an approach that integrates model and preprocessing selection, an experimental framework, evaluation and comparison, and the construction of a stacking ensemble.
```